lib/tasks/thorfile.rb

```diff
@@ -113,6 +113,37 @@ def seed
       puts "[  DONE  ]"
     end
 
+    desc "welcome WELCOME_PACKAGE", "import files and projects from a specficied welcome package"
```

Comment:
@rachkor questions:

- Is there anything else we'd like to do as part of setting up the Welcome?

- How much of a difference would make to have the Welcome be a `.tar.gz` file? Alternatively we can make the DUP's welcome a `.zip` if we think Zip are more user friendly.